hack/jenkins/e2e.sh

```diff
@@ -236,7 +236,11 @@ fi
 # Jenkins will look at the junit*.xml files for test failures, so don't exit
 # with a nonzero error code if it was only tests that failed.
 if [[ "${E2E_TEST,,}" == "true" ]]; then
-    go run ./hack/e2e.go ${E2E_OPT} -v --test --test_args="${GINKGO_TEST_ARGS} --ginkgo.noColor" || true
+    local exitcode
+    go run ./hack/e2e.go ${E2E_OPT} -v --test --test_args="${GINKGO_TEST_ARGS} --ginkgo.noColor" && exitcode=0 || exitcode=$?
+    if [[ $E2E_PUBLISH_GREEN_VERSION == "true" && exitcode == 0 && -z $githash ]]; then
+        echo $githash | gsutil cp -I gs://kubernetes-release/ci/latest-green.txt
+    fi
```

Comment:
@ixdy this is how I prupose publishing the latest-green.txt version. I've add `export E2E_PUBLISH_GREEN_VERSION=true` to the `kubernetes-gce-e2e` job but it's difficult to test. My questions are:
1) Is this the place to do this?
2) Does jenkins have permission to do this?